organic_scoring/organic_queue/organic_queue_base.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+
+class OrganicQueueBase(ABC):
+    """Base organic queue
+    
+    The following methods must be implemented:
+        - add: Add the sample to the queue;
+        - sample: Pop the sample from the queue;
+        - size: Return the size of the queue.
+    """
+
+    @abstractmethod
+    def add(self, sample: Any):
+        """Add the sample to the queue"""
+        raise NotImplementedError
+
+    @abstractmethod
+    def sample(self) -> Any:
+        """Pop the sample from the queue"""
+        raise NotImplementedError
+
+    def size(self) -> int:
```

Comment:
should size also be an abstractmethod?
```suggestion
    @abstractmethod
    def size(self) -> int:
```